board/main.c

```diff
@@ -3,6 +3,11 @@
 
 // ********************* Includes *********************
 #include "config.h"
+#ifdef STM32F4
```

Comment:
I'd put this ifdef in its own header then include that header instead of duplicating this everywhere

Reply:
Noted, maybe even will add this ifdef to config.h, will check, thx